src/imitation/scripts/config/train_adversarial.py

```diff
@@ -145,51 +139,58 @@ def cartpole():
   )
 
 
+@train_ex.named_config
+def mountain_car():
+  env_name = "MountainCar-v0"
+  rollout_hint = "mountain_car"
+
+
+@train_ex.named_config
+def pendulum():
+  env_name = "Pendulum-v0"
+  rollout_hint = "pendulum"
+
+
+# Standard MuJoCo Gym environment named configs
+
+@train_ex.named_config
+def ant():
+  locals().update(**mujoco_shared_locals)
+  env_name = "Ant-v2"
+  rollout_hint = "ant"
+  locals().update(**ant_shared_locals)
```

Comment:
Nit: have both `locals().update` next to each other at the top (otherwise it makes it seem like `ant_shared_locals` overrides `env_name` or `rollout_hint`, which it doesn't).